lib/git-hooks.js

```diff
@@ -131,14 +146,18 @@ function runHooks(hooks, args, callback) {
         return;
     }
 
-    var hook = spawnHook(hooks.shift(), args);
-    hook.on('close', function (code) {
-        if (code === 0) {
-            runHooks(hooks, args, callback);
-        } else {
-            callback(code);
-        }
-    });
+    try {
```

Comment:
i removed it because https://github.com/tarmolov/git-hooks-js/pull/14#discussion_r53565778